assets/section-email-signup-banner.css

```diff
@@ -43,6 +43,10 @@
     color: rgb(var(--color-base-text));
   }
 
+  .banner--desktop-transparent .email-signup-banner__box .field__input {
```

Comment:
Everything works with: `  .banner--desktop-transparent .field__input` but I chose to go with: `.banner--desktop-transparent .email-signup-banner__box .field__input` for better readability. And this only increases the specificity from `(0, 2, 0)` to `(0, 3, 0)` so I'm thinking this is negligible.